src/Admin/FieldDescription.php

```diff
@@ -35,6 +30,7 @@ public function setAssociationMapping($associationMapping)
 
         $this->type = $this->type ?: $associationMapping['type'];
```

Comment:
Off topic: if these ternaries are intended to "catch" only null values (instead of other falsy values), we should use `??` instead.